docs/ossm/istioctl/README.md

```diff
@@ -0,0 +1,70 @@
+[Return to OSSM Docs](../)
+
+# Installing the istioctl tool
+
+The `istioctl` tool is a configuration command line utility that allows service 
+operators to debug and diagnose Istio service mesh deployments.
+
+## Prerequisites
+
+* Use an `istioctl` version that is the same version as the Istio control plane 
+for the Service Mesh deployment. For checking the `Istio` version, run the following command:
+
+    ```bash
+    $ kubectl -n istio-system get istio
```

Comment:
This will go into product doc so should probably use 'oc'

Reply:
Done